ENIGMAsystem/SHELL/Graphics_Systems/General/GSfont.cpp

```diff
@@ -1287,28 +1261,34 @@ void draw_text_ext_color(gs_scalar x, gs_scalar y,variant vstr,gs_scalar sep, gs
   }
 }
 
-unsigned font_get_texture(int fnt) {
-  get_font_null(f,fnt,-1);
-  return f.texture;
+unsigned font_get_texture(int id) {
+  const SpriteFont& fnt = sprite_fonts.get(id);
+  return fnt.texture;
 }
```

Comment:
The following shows `<undefined>` and `0` for GM8.1 and GMSv1.4 in the creation event. So I am totally fine with this, everything that take a font, doesn't accept anything but user fonts. I'm totally fine with that.

GMSv1.4 shows this debug message for -2, -1, and 0 fonts. So we're doing the correct thing here now.
![Font Doesn't Exist](https://user-images.githubusercontent.com/3212801/89819075-d48e1200-db18-11ea-91a7-f7de497f8fde.png)


```cpp
show_message(font_get_name(-2));
show_message(font_get_name(-1));
show_message(font_get_name(0));

show_message(font_get_texture(-2));
show_message(font_get_texture(-1));
show_message(font_get_texture(0));
```